docs/release-notes/march-2024.md

```diff
@@ -0,0 +1,119 @@
+---
+slug: march-2024
+hide_title: true
+title: March 2024
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for March 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in September 2023:
+  * [<i class="fas fa-external-link fa-xs"></i>1.64.0](https://github.com/semgrep/semgrep/releases/tag/v1.64.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.65.0](https://github.com/semgrep/semgrep/releases/tag/v1.65.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.66.0](https://github.com/semgrep/semgrep/releases/tag/v1.66.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.66.1](https://github.com/semgrep/semgrep/releases/tag/v1.66.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.66.2](https://github.com/semgrep/semgrep/releases/tag/v1.66.2)
+  * [<i class="fas fa-external-link fa-xs"></i>1.67.0](https://github.com/semgrep/semgrep/releases/tag/v1.67.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+### Changed
+
+### Fixed
+
+- Visual Studio Code extension: fixed issue where rules weren't downloaded, leading to no scan results.
+
+## 💻 Code
+
+### Added
+
+- Added support for Python's `yield` keyword, enabling the detection of taint findings from taint sources returned by `yield`.
+- Added ability for users to copy file paths displayed in Semgrep Cloud Platform's **Findings** page if they aren't links.
+- Added the ability for users to see if there's a version of a rule they're currently using that supports interfile analysis.
+- **API**: added ability to get rules metadata from the API.
+
+### Changed
+
+- Code analysis started by logged-in users running `semgrep ci` now includes cross-file (interfile) analysis.
+- `.phtml` files are now processed as PHP files and analyzed using PHP rules.
+- Updated PR comments to include links to specific findings in Semgrep Cloud Platform.
+- Users can see all projects, even if they don't have any identified findings, in Semgrep Cloud Platform.
```

Comment:
They should have already been able to see all projects in **Projects**. What's changed here?